packages/components/src/spinner/style.scss

```diff
@@ -0,0 +1,33 @@
+@keyframes rotation {
```

Comment:
Should we rename this `components-spinner-rotation` or something like that to avoid ambiguity. Alternatively we can move this animation to `_animations` because it looks generic enough

Reply:
Agreed! I've moved it to `_animations`.